tests/unit/info.tcl

```diff
@@ -391,7 +391,13 @@ start_server {tags {"info" "external:skip" "debug_defrag:skip"}} {
             # set qbuf limit to minimum to test stat
             set org_qbuf_limit [lindex [r config get client-query-buffer-limit] 1]
             r config set client-query-buffer-limit 1048576
-            catch {r set key [string repeat a 1048576]}
+            catch {r set key [string repeat a 2048576]} e
```

Comment:
Why did we almost double the amount of data to be written? And what's the reason for such errors (is it well understood)?

Reply:
Yeah, this just makes the failure mode I discuss happen more consistently since we have lots more to write after th server disconnects. So if my change doesn't work somewhere it'll be obvious.